### PR TITLE
Memory Pair Game homework

### PR DESCRIPTION
**Pull request by Roman-Halenko**
### Memory Pair Game
[Demo](https://roman-halenko.github.io/memory-pair-game/) |
[Code base](https://github.com/Roman-Halenko/memory-pair-game)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Roman-Halenko/memory-pair-game/game.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+         <div class="front">
+         </div>
+       </div>
+     </div>`)
+};
+
+// Fill the area with cards
+function fillArea(src) {
+  let somthing = src.concat(src);
+  somthing.sort( () => { return 0.5 - Math.random() } ).forEach(addCard);
```
[the remark this answers is not in the document]
Comment (Roman-Halenko) [reply submissions/Roman-Halenko/memory-pair-game/game.js]: Course, as arrow function body has a single statement, we may get rids of curly brackets.

What a shame, <code> somthing</code>... I've forgotten to give a smarter name to a variable 😳

File: submissions/Roman-Halenko/memory-pair-game/game.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      card.classList.add('hidden');
+    }
+  })
+};
+
+function congrats() {
+  let cards = document.querySelectorAll('.flipper-container');
+  let hidden = document.querySelectorAll('.hidden');
+  if (hidden.length === cards.length) {
+    alert('Wubba Lubba Dub Dub!');
```
[the remark this answers is not in the document]
Comment (Roman-Halenko) [reply submissions/Roman-Halenko/memory-pair-game/game.js]: 😁😁

**Comment by Roman-Halenko** [comment]
@zonzujiro 
> Also, IMO, what if you will use https://rickandmortyapi.com/ for next project 😉

Awesome!!

File: submissions/Roman-Halenko/memory-pair-game/game.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  }
+};
+
+function update() {
+  let openedCards = document.querySelectorAll('.open');
+
+  if (openedCards.length === 2) {
+    if (openedCards[0].isEqualNode(openedCards[1])) {
+      setTimeout(moveUp, 700, openedCards);
```
[the remark this answers is not in the document]
Comment (Roman-Halenko) [reply submissions/Roman-Halenko/memory-pair-game/game.js]: I got rid of some magic numbers. But I'm not sure that I implemented in the best way.

**Comment by zonzujiro** [comment]
Done. Comments FYI

**Pull request merged**